projects/js-packages/charts/src/types.ts

```diff
@@ -107,6 +107,21 @@ export type ChartTheme = {
 	/** Styles for legend container */
 	legendContainerStyles?: CSSProperties;
 	annotationStyles?: AnnotationStyles;
+	/** LeaderboardChart specific settings */
+	leaderboardChart?: {
```

Comment:
I'm not very sure of adding all these to the theme. Curious to know what @retrofox thinks.

Reply:
I agree that the current usage would be pretty specific. There is `DEFAULT_LEADERBOARD_SETTINGS` for the component default settings, so the theme settings could be removed until other styling is needed. Will follow up on another PR.